### PR TITLE
only produce error events for system pods, not e2e pods

### PR DESCRIPTION
**Pull request by deads2k**
This will prune the list of hundreds of failing events on the monitor down to something managable.  Not all e2e pods are even expected to run, so having some fail isn't something to fail or flake on.

### CODE REVIEW

File: pkg/monitor/pod.go [hunk 1]
```diff
@@ -73,6 +73,14 @@ func startPodMonitoring(ctx context.Context, m Recorder, client kubernetes.Inter
 	podChangeFns := []func(pod, oldPod *corev1.Pod) []Condition{
 		// check phase transitions
 		func(pod, oldPod *corev1.Pod) []Condition {
+			podErrorLevel := Error
+			isSystemPod := filterToSystemNamespaces(pod)
```
Comment (bparees) [inline pkg/monitor/pod.go:77]: how is this defined?  openshift-*?

Comment (bparees) [reply pkg/monitor/pod.go:77]: ah, found it.

**Comment by deads2k** [comment]
huh, I hope those weren't me

/test all

**Comment by smarterclayton** [comment]
The problem here is that when low level stuff dies, it dies and leaves events that are useful and searchable in this list.  Ie any of the “container runtime failed to start” or “networking choked”.  Right now those are included because we had 2-3 back to back that showed them.

It might be better to prune a specific list of predictable events.  Can we try to identify a list of those first and see the number?

/hold

**Comment by smarterclayton** [comment]
Another option is to dump them in another junit (split in the monitor based on namespace starting with e2e).

**Comment by deads2k** [comment]
> The problem here is that when low level stuff dies, it dies and leaves events that are useful and searchable in this list. Ie any of the “container runtime failed to start” or “networking choked”. Right now those are included because we had 2-3 back to back that showed them.
> 
> It might be better to prune a specific list of predictable events. Can we try to identify a list of those first and see the number?
> 
> /hold

This PR still allows them to come out. They come out as warnings.

**Pull request closed without merging**